### PR TITLE
Fixing AssemblyNames in a few test projects

### PR DESCRIPTION
**Pull request by joperezr**
cc: @weshaggard  @MattGal 

Fixing the rest of the projects where AssemblyName doesn't match the project name.

### CODE REVIEW

**Comment by MattGal** [comment]
LGTM.  From the non-included changes, it might be useful to run your tool just to load the project and write it back to get uniform formatting at some point.

**Comment by joperezr** [comment]
> it might be useful to run your tool just to load the project and write it back to get uniform formatting at some point.

Sure, I think we should  do that in a different PR though.

File: src/System.Private.Uri/tests/FunctionalTests/UriParserTest.cs [hunk 1]
```diff
@@ -193,6 +193,7 @@ public static void GetComponents_Null()
         [Fact]
         public static void GetComponents_BadUriComponents()
         {
+            http = new Uri(full_http);
```
Comment (weshaggard) [inline src/System.Private.Uri/tests/FunctionalTests/UriParserTest.cs]: This looks unrelated. Is it the produce of a merge?

**Comment by weshaggard** [comment]
> it might be useful to run your tool on every project just to load the project and write it back to get uniform formatting at some point.

I would prefer the tool simply remove the AssemblyName property from the projects, especially for test projects where it isn't interesting, but I think we could safely remove it from every project. I'm not aware of any current projects in CoreFx where the project and assembly name are intentionally different.

**Comment by MattGal** [comment]
Joe's tool did find some where the Assembly name intentionally pruned "Microsoft. ... " to have a shorter name than the project;  none that seem like they'd break though.

File: src/System.Private.Uri/tests/FunctionalTests/UriParserTest.cs (hunk 1 above)
Comment (MattGal) [inline src/System.Private.Uri/tests/FunctionalTests/UriParserTest.cs]: I asked about it yesterday ... there was something about renaming the assembly exposed a bug with the uninitialized variable by changing test execution ordering.  It's a voodoo change but it checks out.
